:160000 160000 994aa6e 994aa6e M	ext/glad

Submodule ext/glad contains modified content
diff --git a/ext/glad/cmake/CMakeLists.txt b/ext/glad/cmake/CMakeLists.txt
index b395f53..8648c4b 100644
--- a/ext/glad/cmake/CMakeLists.txt
+++ b/ext/glad/cmake/CMakeLists.txt
@@ -264,11 +264,12 @@ function(glad_add_library TARGET)
     add_library("${TARGET}" ${GLAD_ADD_LIBRARY_ARGS}
         ${GLAD_FILES}
         )
-
-    target_include_directories("${TARGET}"
-        PUBLIC
-            "${GLAD_DIR}/include"
-        )
+    target_include_directories(
+            "${TARGET}"
+            PUBLIC
+            $<BUILD_INTERFACE:${GLAD_DIR}/include>
+            $<INSTALL_INTERFACE:include/>
+    )
 
     target_link_libraries("${TARGET}"
         PUBLIC
